specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-11-15/mongorbac.json

```diff
@@ -406,7 +406,7 @@
           "description": "A user-friendly name for the Role Definition. Must be unique for the database account."
         },
         "type": {
-          "type": "string",
+          "type": "int",
           "enum": [
             "BuiltInRole",
```

Comment:
Does it have default value? If yes, does it need to add ""default": 1," or ""default": 0," like this [example](https://github.com/Azure/azure-rest-api-specs/blob/969fd0c2634fbcc1975d7abe3749330a5145a97c/specification/mixedreality/resource-manager/Microsoft.MixedReality/stable/2021-01-01/common.json#L38)?

Reply:
fixed